aws-aurora-postgres/main.tf

```diff
@@ -0,0 +1,87 @@
+locals {
+  name = "${var.project}-${var.env}-${var.service}"
+
+  tags = {
+    managedBy = "terraform"
+    Name      = "${var.project}-${var.env}-${var.service}"
+    project   = "${var.project}"
+    env       = "${var.env}"
+    service   = "${var.service}"
+    owner     = "${var.owner}"
+  }
+}
+
+resource "aws_security_group" "rds" {
+  name        = "${local.name}"
+  description = "Allow postgres inbound"
+
+  vpc_id = "${var.vpc_id}"
+
+  ingress {
+    from_port   = "${var.database_port}"
```

Comment:
hmm maybe these shouldn't be vars. not currently propagated to the db resource.

Reply:
That's weird, not sure how that would have worked before unless no one set them.